### PR TITLE
feat(collections): add fromItems and containsKey

### PR DESCRIPTION
**Pull request by pfmmfp**
- Adding `*::fromItems()` to allow instantiating a collections from any iterable.
- Adding `*->containsKey()` as an alias of `contains`

### CODE REVIEW

File: src/Psl/Collection/MutableVector.php [hunk 1]
```diff
@@ -69,6 +70,20 @@ public static function fromArray(array $elements): MutableVector
...
+    /**
+     * Create a vector from the given $items iterable.
+     *
+     * @template Ts
+     *
+     * @param iterable<Ts> $items
+     *
+     * @return MutableVector<Ts>
+     */
+    public static function fromItems(iterable $items): MutableVector
```
Comment (azjezz) [inline src/Psl/Collection/MutableVector.php:82]: could you please add the same method to other collection types? ( `Vector`, `Set`, `MutableSet`, `Map`, and `MutableMap` ).

**Review by azjezz — DISMISSED** [review]

File: src/Psl/Collection/MutableVector.php (hunk 1 above)
Comment (veewee) [reply src/Psl/Collection/MutableVector.php:82]: Maybe `fromIterable()` would be more meaningful?

Comment (pfmmfp) [reply src/Psl/Collection/MutableVector.php:82]: Done! Also added an alias for `containsKey` for parity with hhvm collections

File: src/Psl/Collection/IndexAccessInterface.php [hunk 2]
```diff
@@ -34,6 +34,17 @@ public function at(int|string $k): mixed;
...
+    /**
+     * Alias of `contains`.
+     *
+     * @param Tk $k
+     *
+     * @see contains() method
+     *
+     * @psalm-mutation-free
+     */
+    public function containsKey(int|string $k): bool;
```
Comment (veewee) [inline src/Psl/Collection/IndexAccessInterface.php:46]: This new method is not covered by the test-suite.

Comment (pfmmfp) [reply src/Psl/Collection/IndexAccessInterface.php:46]: Added. Thanks for the hint!

Comment (veewee) [reply src/Psl/Collection/IndexAccessInterface.php:46]: Thanks. There are still some generated documentation and psalm issues left.
The infection task is fixed in a separate PR and is not related to this change.

Comment (veewee) [reply src/Psl/Collection/IndexAccessInterface.php:46]: Almost there ... :)

```sh
make coding-standard-fix
make docs-generate
```

**Review by veewee — DISMISSED** [review]

**Review by veewee — APPROVED** [review]

File: src/Psl/Collection/IndexAccessInterface.php (hunk 2 above)
Comment (veewee) [reply src/Psl/Collection/IndexAccessInterface.php:46]: Thanks!

**Pull request merged**